cleanlab/object_detection/rank.py

```diff
@@ -0,0 +1,1157 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+import warnings
+from typing import List, Dict, Any, Optional, Tuple
+import numpy as np
+import copy
+
+# for MAP
+from multiprocessing import Pool
+
+"""Methods to rank and score images in an object detection dataset (object detection data), based on how likely they
+are to contain label errors. """
+
+
+def get_label_quality_scores(
+    annotations: List[Dict[Any, Any]],
+    results: List[np.ndarray],
+    *,
+    method: str = "map",
+    threshold: Optional[float] = None,
+    verbose: bool = True,
+) -> np.ndarray:
+    """Returns a label quality score for each datapoint.
+
+    This is a function to compute label quality scores for object detection (multi-class) datasets,
```

Comment:
Are we sticking with naming convention: "object detection (multi-class) datasets" ?

Can you link to where that convention comes from?